cli/Cargo.toml

```diff
@@ -39,4 +39,8 @@ colored = "2.1.0"
 const_format = "0.2.32"
 hpke-dispatch = { version = "0.5.1", features = ["serde"], optional = true }
 rand = { version = "0.8.5", optional = true }
+janus_client = "0.7.7"
```

Comment:
Hmm. I had punted on a tool like this before because supporting multiple DAP versions is a somewhat tricky problem. I think for now, we can get away with supporting only DAP-09, since the other "supported" versions are on their way out.